contentcuration/contentcuration/viewsets/feedback.py

```diff
@@ -0,0 +1,53 @@
+from rest_framework import permissions
+from rest_framework import serializers
+from rest_framework import viewsets
+from rest_framework.permissions import IsAuthenticated
+
+from contentcuration.models import BaseFeedback
+from contentcuration.models import BaseFeedbackEvent
+from contentcuration.models import BaseFeedbackInteractionEvent
+from contentcuration.models import FlagFeedbackEvent
+
+
+class IsAdminForListAndDestroy(permissions.BasePermission):
+    def has_permission(self, request, view):
+        # only allow list and destroy of flagged content to admins
+        if view.action in ['list', 'destroy']:
+            try:
+                return request.user and request.user.is_admin
+            except AttributeError:
+                return False
+        if request.user.check_feature_flag('test_dev_feature'):
+            return True
+        return False
+
+
+class BaseFeedbackSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = BaseFeedback
```

Comment:
I'm wondering if there is any benefit to referencing the model in these base serializers? since it just gets overridden when inherited, and the overridden model would inherit the base model, it should be no different?